Quick note about CSV downloads (#246)
getredash/website: src/pages/kb/user-guide/querying/download-query-results.md

- Query API key.
- Latest results URL in CSV and JSON format. You can also get the Excel format by changing the file type suffix from `json`/`csv` to `xlsx`.

{% callout info %}

> The latest results in CSV or JSON format option will not work for queries that use parameters.

Reviewer: ```suggestion
The latest results API is not supported for queries that use parameters.
```
